packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,351 @@
+                width: 100%;
+                height: 74px;
+                border-top: 1px solid #F0F0F1;
+            }
+
+        }
+
+        .confirm-txt {
+            font-size: 14px;
+            line-height: 150%;
+            color: #72727A;
+            margin-left: 16px;
+        }
+
+
+        .h-right {
+            position: absolute;
+            right: 16px;
+            text-align: right;
+        }
+
+        .estimated-fees {
+            width: 100%;
+            margin-top: 20px;
+        }
+
+        .account-id {
+            font-weight: bold;
+            font-size: 14px;
+            line-height: 150%;
+            text-align: right;
+            color: #272729;
+        }
+
+        h3 {
+            font-weight: 900;
+            font-size: 20px;
+            line-height: 130%;
+            text-align: center;
+            color: #24272A;
+        }
+
+        .success {
+            > p {
+                font-weight: 500;
+                font-size: 20px;
+                line-height: 150%;
+                text-align: center;
+                color: #3F4045;
+                margin-bottom: 0px;
+            }
+        }
+
+        p {
+            font-weight: 500;
+            font-size: 14px;
+            line-height: 150%;
+            text-align: center;
+            color: #72727A;
+        }
+
+        .receiver-input {
+            width: 100%;
+        }
+
+        .amount-grey {
+            font-weight: 500;
+            font-size: 14px;
+            line-height: 150%;
+            color: #A2A2A8;
+        }
+
+        .v-center {
+            position: absolute;
+            top: 50%;
+            transform: translate(0, -50%);
+        }
+
+        .icon {
+            margin-bottom: 20px;
+        }
+
+        form {
+            width: 100%;
+        }
+
+        .full-width {
+            width: 100%;
+        }
+
+        .modal-footer {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            button {
+                flex: 1;
+                margin: 0 20px;
+            }
+        }
+    }
+`;
+
+export default function NFTTransferModal({ open, onClose, nft, accountId }) {
+    const [receiverId, setReceiverId] = useState('');
+    const [result, setResult] = useState();
+    const [sending, setSending] = useState(false);
+    const [viewType, setViewType] = useState('transfer');
+    const [accountIdIsValid, setAccountIdIsValid] = useState(false);
+    const balance = useSelector(state => selectBalance(state));
+    const nearBalance = balance.balanceAvailable;
+    const dispatch = useDispatch();
+    const { updateNFTs } = nftActions;
+
+    const { localAlert } = useSelector(({ status }) => status);
+
+    function onTransferSuccess(result) {
+        setResult(result);
+        dispatch(updateNFTs({ accountId }));
+        setViewType('success');
+    }
+
+    async function sendNFT(nft, receiverId) {
```

Comment:
Generally, avoid referencing some data from closure-scope and other data as arguments if all of the data is in the parent closure scope.  :hocho: either remove these arguments (just reference the variables in the parent closure scope directly and don't shadow those variables), or pass _all_ of the variables referenced in this function into the function as arguments (with names that do not shadow variable names in the parent closure)